### DISCUSSION TITLE
Additional Filters

### DISCUSSION THREAD

**Discussion by SJPadbury** [sct/overseerr#1371] [Feature Requests]
Couple of "nice to haves":
  When looking at search results or actor pages being able to filter out items that already are added to the library.
  Also would be nice to be able to look at just an actor's Movie or just their TV Series.
  Finally, being able to filter out appearances as "self" would be nice, to reduce the fact that any major actor/actress comes up as being in every late night show on the planet.

**Comment by sct** [comment]
We intentionally don't filter out available items when searching. It would be confusing to search for something and not be able to find it.

**Comment by SJPadbury** [comment]
I completely understand where you're coming from there, but maybe default result of show everything, but you can click a checkbox each time to filter it down?

**Reply by sct** [reply]
Then you will want to follow this issue: https://github.com/sct/overseerr/issues/459

**Reply by SJPadbury** [reply]
Ooh, I completely missed that when searching before creating. (Filters vs Filtering, I guess.)
Thanks!

**Comment by SJPadbury** [comment]
So looking at that issue and the associated feature https://github.com/sct/overseerr/pull/3232 from a couple years ago, I see it was implemented, although in a limited fashion only on the direct search?

What I'd really love to see is for this filtering to have wider use. Like, the Trending/Popular screens, or an individual actor's page, as well as adding the ability to filter out as I said before, items we already have, which I don't see an issue with including the option if every page's first load is unfiltered.
